lib/helpers.bash

```diff
@@ -456,6 +574,25 @@ _bash-it-reload() {
   popd &> /dev/null || return
 }
 
+_bash-it-process-component ()
```

Comment:
Can you think of a more descriptive name for this function?

Reply:
take a look now